Library/Formula/freeswitch.rb

```diff
@@ -19,20 +19,24 @@ class Freeswitch < Formula
   depends_on "pkg-config" => :build
   depends_on :apr => :build
 
-  depends_on "jpeg"
+  option "without-moh", "Do not install music-on-hold"
+  option "without-sounds-en", "Do not install English (Callie) sounds"
+  option "with-sounds-fr", "Install French (June) sounds"
+  option "with-sounds-ru", "Install Russian (Elena) sounds"
+
   depends_on "curl"
+  depends_on "jpeg"
+  depends_on "ldns"
   depends_on "openssl"
   depends_on "pcre"
   depends_on "speex"
   depends_on "sqlite"
 
   def install
-    system "./bootstrap.sh", "-j#{ENV.make_jobs}"
+    system "./bootstrap.sh", "-j"
```

Comment:
Yeah, looks like that's true: https://freeswitch.org/stash/projects/FS/repos/freeswitch/browse/bootstrap.sh#22